Gruntfile.js

```diff
@@ -33,44 +35,8 @@ module.exports = function(grunt) {
       files: ["tpl/programs.json"]
     },
     nodeunit: {
-      tests: [
-        "test/bootstrap.js",
-        "test/board.js",
-        "test/board-connection.js",
-        "test/compass.js",
-        "test/options.js",
-        "test/board.pins.js",
-        "test/board.component.js",
-        "test/capabilities.js",
-        // ------------------
-        "test/accelerometer.js",
-        "test/animation.js",
-        "test/button.js",
-        "test/esc.js",
-        "test/fn.js",
-        "test/gyro.js",
-        "test/imu.js",
-        "test/lcd.js",
-        "test/led.js",
-        "test/ledcontrol.js",
-        "test/motor.js",
-        "test/pin.js",
-        "test/piezo.js",
-        "test/ping.js",
-        "test/pir.js",
-        "test/proximity.js",
-        "test/reflectancearray.js",
-        "test/relay.js",
-        "test/repl.js",
-        "test/sensor.js",
-        "test/servo.js",
-        "test/shiftregister.js",
-        "test/sonar.js",
-        "test/stepper.js",
-        "test/temperature.js",
-        "test/switch.js",
-        "test/wii.js"
-      ]
+      // remove test/distance.js (and this exception) when bumping to 0.9.0
+      tests: ["test/*.js", "!test/mock*", "!test/distance.js"]
```

Comment:
I appreciate what you've done here, but I have about 30 untracked files in the test/ dir of my local repo that shouldn't be run—unless you know of a good way around that?